app/controllers/reviewers/applications_controller.rb

```diff
@@ -68,6 +69,10 @@ def store_filters
       end
     end
 
+    def mentor_comments
+      Mentor::Comment.where(commentable_id: @application.id).where('created_at > ?', Date.parse('2018.03.12'))
```

Comment:
Maybe I'm missing context, but what is the condition on `created_at` for? And why is it hardcoded to a certain value...?

Reply:
Good question! This comes from @carpodaster 's review [here](https://github.com/rails-girls-summer-of-code/rgsoc-teams/pull/965#pullrequestreview-102945776) where it makes sense to only load recent mentor reviews, as previously these were private.

Reply:
😹 ah ok, now I get it, thanx!